### DISCUSSION TITLE
Welcome to SCAS Discussions!

### DISCUSSION THREAD

**Discussion by babkiniaa** [babkiniaa/SCAS#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ShchekoturovDA** [comment]
Privet

**Reply by babkiniaa** [reply]
hi

**Comment by babkiniaa** [comment]
Можете предлагать свои идеи мне в личку в Дискорде!

**Reply by ShchekoturovDA** [reply]
Ссылку дай

**Reply by babkiniaa** [reply]
Пока зайти не могу, позже скину

**Reply by babkiniaa** [reply]
Пока можешь написать мне в Instagram!
